tox.ini

```diff
@@ -0,0 +1,46 @@
+[tox]
+isolated_build = true
+minversion: 3.14.1
```

Comment:
Here and on line 44 `:` is used instead of `=` as the kv separator. It would be good to be uniform.

Reply:
Good catch. 

(I really don't like ini..there's no standard for it and so many "gotchas". One of my side projects coming up is to add support for `pants.toml`.)